proposals/fleet-multiple-abm-vpp.md

```diff
@@ -0,0 +1,130 @@
+### Multiple ABM/VPP tokens
+
+- Figma: https://www.figma.com/design/j2M1heOh8eZD6LcUJks6HE/%239956-Add-multiple-Apple-Business-Manager-and-Volume-Purchasing-Program-connections?node-id=2-130&t=3vPRYgndVgzoD8e4-0
+- Issue: https://github.com/fleetdm/fleet/issues/9956
+
+- Constraints/bussines logic:
+    - VPP tokens can be assigned to: "All teams", "No team" or a team. Default is "All teams"
+    - ABM tokens can be assigned to: "No team" or a team. Default is "No team"
+    - Multiple ABM/VPP tokens per team.
+    - Same private key can be used for multiple tokens.
```

Comment:
What private key are we talking about here? `mdm.apple_bm_key`? So the same Apple BM cert and key can be used (MUST be used as we don't add config options?) to generate all distinct tokens?

Reply:
> What private key are we talking about here? mdm.apple_bm_key? 

exactly! and this is just a simplification because we could create a new key every time the endpoint is hit, I didn't see any concerns sharing it to generate all distinct tokens, but this part is not 100% clear to me yet.

Reply:
@roperzh will we still use `server_private_key` to encrypt the `vpp_token`s?

Reply:
@jahzielv that's the plan! do you see any problems with that?

Reply:
> that's the plan! do you see any problems with that?

Nope, lgtm!